Classes/Backend/TceMain.php

```diff
@@ -268,7 +268,7 @@ public function processCmdmap_postProcess(&$command, $table, $id, &$relativeTo,
         $record = $this->resolveRecordForOperation($table, $id);
 
         if ($table === 'tt_content') {
-            if ('localize' === $command) {
+            if ('localize' === $command || 'move' === $command) {
```

Comment:
I'm not sure this is good - the method that gets called here is quite heavy on SQL requests and we don't want that executing every time you drag and drop anything. This should absolutely have a guard clause to make sure it doesn't get called if there is no reason to call it. That could be done with early returns from `fixPositionInLocalization` so you don't have to, for example, fetch a record here.